libs/ob-sql-parser/src/main/resources/oboracle/sql/OBParser.g4

```diff
@@ -1444,10 +1481,20 @@ trans_param_value
     | INTNUM
     ;
 
+dblink_info_param_name
+    : STRING_VALUE
+    ;
+
+dblink_info_param_value
```

Comment:
these tow expression are never used, we may delete it

Reply:
done